bundles/org.openhab.binding.denonmarantz/README.md

```diff
@@ -118,7 +118,7 @@ Group item=marantz_input label="Receiver" icon="receiver" {
 
 These resources can be useful to learn what to send using the `command`channel:
 
-- [AVR-X2000/E400](http://www2.aerne.com/Public/dok-sw.nsf/0c6187bc750a16fcc1256e3c005a9740/96a2ba120706d10dc1257bdd0033493f/$FILE/AVRX2000_E400_PROTOCOL(10.1.0)_V04.pdf)
+- [AVR-X2000/E400](https://assets.denon.com/documentmaster/uk/avrx2000_e400_protocol(1010)_v03.pdf)
```

Comment:
Not sure if it makes any difference, but this lnks to V03 and the broken link refers to V04

Reply:
To be honest, I don't know either, if it makes a difference. I can't compare the documents as I don't have the V04. The commands (starting from page 8) work for me. So my assumption was, an old version is better than no version or a broken link.

Reply:
I think "V04" in the filename might have been wrong. At least it's confusing, since the newly linked PDF is "Rev 10.1.0" and "Ver 03". The old filename referred also to "10.1.0" but to "V04". The new link refers to an asset on Denon's own site. We can't compare the documents, but it looks to be as good as it gets.

The link below is also broken btw.